### PR TITLE
Fixed an issue were doc_images required release-notes and validations

### PR DESCRIPTION
**Pull request by bakatzir**
## Status
- [x] Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/28332

## Must have
- [x] Tests

### CODE REVIEW

File: demisto_sdk/commands/update_release_notes/update_rn.py [hunk 1]
```diff
@@ -153,9 +153,10 @@ def is_bump_required(self):
             sys.exit(0)
         return True
 
-    def only_readme_changed(self):
+    def only_docs_changed(self):
         changed_files = self.added_files.union(self.modified_files_in_pack)
-        if len(changed_files) == 1 and 'README' in changed_files.pop():
+        if (len(changed_files) == 1 and 'README' in changed_files.pop()) or \
+                (all('README' in file or '.png' in file for file in changed_files)):
```
Comment (roysagi) [inline demisto_sdk/commands/update_release_notes/update_rn.py:159]: this is not good as non-doc images might be here...maybe use find_type to check for README/DOC_IMAGE?

File: demisto_sdk/commands/validate/validate_manager.py [hunk 2]
```diff
@@ -93,7 +93,7 @@ def __init__(self, is_backward_check=True, prev_ver=None, use_git=False, only_co
                                    FileType.DESCRIPTION,
                                    FileType.TEST_PLAYBOOK,
                                    FileType.TEST_SCRIPT,
-                                   )
```
Comment (roysagi) [inline demisto_sdk/commands/validate/validate_manager.py:96]: we also need to ignore this in RN check - look at `validate_no_missing_release_notes` - add it to the ignored files there

File: demisto_sdk/commands/update_release_notes/update_rn.py (hunk 1 above)
Comment (bakatzir) [reply demisto_sdk/commands/update_release_notes/update_rn.py:159]: Tried, but tests are heavily dependent on paths(and not real files)
I'll try something else.

**Review by roysagi — APPROVED** [review]
seems good, please run update-rn on an existing pack where you add a doc file - just to make sure it works

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,4 +1,5 @@
 # Changelog
+* Fixed an issue were doc_images required release-notes and validations
```
Comment (roysagi) [inline CHANGELOG.md:2]:
```suggestion
* Fixed an issue were doc_images required release-notes and validations.
```

**Comment by bakatzir** [comment]
Screen Shot 2020-09-10 at 15 34 18

```
/Users/bkatzir/dev/demisto/demisto-sdk/.tox/py37/bin/python /Users/bkatzir/dev/demisto/demisto-sdk/demisto_sdk/__main__.py update-release-notes -i Packs/ServiceNow/ -u revision
Starting to update release notes.
Collecting all committed files
Collecting all local changed files against the content master
Release notes were found at Packs/ServiceNow/ReleaseNotes/1_2_4.md. Skipping
No changes to pack files were detected from the previous time this command was run. The release notes have not been changed.
```

**Pull request merged**